### DISCUSSION TITLE
Thanks Gerry

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#330] [General]
I would like to thank Gerry for all the time and effort on this project.  This works better than the slingbox software, for my needs.  I have both Plex and VLC access setup and it works flawlessly.  

Best Regards,
Peter

**Comment by blasco403** [comment]
I agree with Peter (pfuller88). Thanks Gerry and all others that have contributed to Slinger.

It works great.

Thanks

Ron

**Comment by robosmith** [comment]
HUGE thanks to @GerryDazoo for resurrecting Slingboxes from the grave in which Dish buried them.

**Comment by glennybob** [comment]
Total agreement here!  Thanks so much Gerry!

**Reply by MikeVTO** [reply]
I echo the sentiments above.  Great work!

**Comment by BobC1954** [comment]
Yes, I also echo thanks to Gerry. I had visions of loosing the use of my Slingboxes, but Gerry's Slinger sorted it!

**Comment by dem5996** [comment]
Ditto - Thank You Gerry !!!
